app/models/region.rb

```diff
@@ -53,6 +53,52 @@ def protected_areas_per_governance
     }
   end
 
+  def protected_areas_per_iucn_category
+    region_data = Hash.new { |hash, key| hash[key] = Hash.new }
```

Comment:
This hash is unnecessary as far as I can see, it just ends up creating an empty hash. Might as well use the literal `region_data = {}`

Reply:
Or do we even need to define empty arrays and hashes? Can we just make them as we go?

Reply:
@spencerldixon I did try making the hash as we go but I got some errors, so I found this solution to the problem on Stack Overflow. I could try and make the change you suggest but I get the following result:

```
undefined method `[]' for nil:NilClass
```

Reply:
instead of that line, have you tried with just `region_data = {}` or did it fail as well?

Reply:
Thanks @Levia I will try that now!

Reply:
When I try that I get:

`undefined method `[]' for nil:NilClass`